docassemble/ALWeaver/data/questions/template_validation.yml

```diff
@@ -677,7 +703,7 @@ code: |
 
   for field in validation_fields:
     if field.source_document_type == "docx":
-      field.reserved = is_reserved_docx_label(field.variable) or field.variable in people_list
+      field.reserved = is_reserved_docx_label(field.variable) or field.variable in people_list or matching_reserved_names({field.variable})
```

Comment:
I thought you were trying to separate out the two concepts of "used in our question library" and "reserved", but this combines them into the same thing?

Reply:
No, this is only for display in the preflight checker